core/src/main/java/org/elasticsearch/common/geo/builders/PolygonBuilder.java

```diff
@@ -141,9 +142,10 @@ protected void validateHole(LineStringBuilder shell, LineStringBuilder hole) {
 
         Edge[] edges = new Edge[numEdges];
         Edge[] holeComponents = new Edge[holes.size()];
-        int offset = createEdges(0, orientation, shell, null, edges, 0);
+        final AtomicBoolean translate = new AtomicBoolean(false);
```

Comment:
Sure, will do the renaming, also in the method arguments. 
I choose AtomicBoolean not for thread safety but because it has updateable state so it can be passed around in the methods and changed. If any wrapper around a primitive boolean exists that is more lightweight I'm happy to change, but couldn't find anything on a quick search.